### PR TITLE
Add my file1

### PR DESCRIPTION
**Pull request by susmitamondal29**
Polarized ZZ: Z0Z0 - Both Z's longitudinally polarized, ZTZT -Both Z's transversely polarized

### CODE REVIEW

**Comment by sihyunjeon** [comment]
@susmitamondal29 Hi,
Please remove *.dat~ files

File: bin/MadGraph5_aMCatNLO/cards/examples/Z0Z0_LO/Z0Z0_LO_run_card.dat [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+# Automatic ptj and mjj cuts if xqcut > 0
+# (turn off for VBF and single top processes)
+#**********************************************************
+   T  = auto_ptj_mjj  ! Automatic setting of ptj and mjj
+#**********************************************************
+#                                                                    
+#**********************************
+# BW cutoff (M+/-bwcutoff*Gamma)
+#**********************************
+  15000  = bwcutoff      ! (M+/-bwcutoff*Gamma)
```
Comment (sihyunjeon) [inline bin/MadGraph5_aMCatNLO/cards/examples/Z0Z0_LO/Z0Z0_LO_run_card.dat:87]: Is there a reason to give bwcutoff this big?

Comment (susmitamondal29) [reply bin/MadGraph5_aMCatNLO/cards/examples/Z0Z0_LO/Z0Z0_LO_run_card.dat]: I have edited the bwcutoff to 15.0 and committed the new changes

**Comment by susmitamondal29** [comment]
> @susmitamondal29 Hi,
> Please remove *.dat~ files

**Comment by agrohsje** [comment]
@susmitamondal29 why are you opening and closing different pr's? i also suggest to give the pull request a proper name so we know what this is about.

**Pull request closed without merging**